startupscript/gcp/vm-metadata.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# vm-metadata.sh
+
+# Retrieves instance attributes set on the VM.
+
+function get_metadata_value() {
+ local metadata_path="${1}"
```

Comment:
The AWS version validates that a parameter is passed. Any reason that these are not consistent in their structure?